README.md

```diff
@@ -1,11 +1,16 @@
-# LaunchDarkly Lua Server-Side SDK HAProxy example
+# LaunchDarkly Lua server-side SDK HAProxy example
 
-This is a minimal dockerized example of using the Lua SDK with [HAProxy](https://www.haproxy.org/).
+We've built a minimal dockerized example of using the Lua SDK with [HAProxy](https://www.haproxy.org/). For more comprehensive instructions, you can visit the [Using the Lua SDK with HAProxy guide](https://docs.launchdarkly.com/guides/sdk/haproxy) or the [Lua reference guide](https://docs.launchdarkly.com/sdk/server-side/lua).
```

Comment:
@mmrj should we still link to tha API docs here?

Reply:
yeah, linking to the SDK reference guide makes sense to me